.env.example

```diff
@@ -64,7 +64,6 @@ OIDC_USERQUERY_OPERATOR=<"or"|"and">
 OIDC_USERQUERY_FILTER="username:username, email:email"
 
 ELASTICSEARCH_ENABLED=<"yes"|"no">
-APP_PORT=3003
```

Comment:
nit: unrelated?

Reply:
This ENV variable is not used anywhere and I felt the change was too small to open a PR..  As it's an unused variable in an example file